src/plugin.c

```diff
@@ -214,9 +471,37 @@ EXPORT void CALL InitiateRSP(RSP_INFO Rsp_Info, unsigned int* UNUSED(CycleCount)
     l_ProcessAlistList = Rsp_Info.ProcessAlistList;
     l_ProcessRdpList = Rsp_Info.ProcessRdpList;
     l_ShowCFB = Rsp_Info.ShowCFB;
+
+    setup_rsp_fallback(ConfigGetParamString(l_ConfigRspHle, RSP_HLE_CONFIG_FALLBACK));
+
+    m64p_rom_header rom_header;
+    CoreDoCommand(M64CMD_ROM_GET_HEADER, sizeof(rom_header), &rom_header);
+
+    /* Init hle_gfx and hle_aud variables - with game-specific tweaks */
+    if ((strstr((char*)rom_header.Name, (const char*)"WORLD DRIVER CHAMP") != NULL)
+     || (strstr((char*)rom_header.Name, (const char*)"Indiana Jones") != NULL)
+     || (strstr((char*)rom_header.Name, (const char*)"Battle for Naboo") != NULL)
+     || (strstr((char*)rom_header.Name, (const char*)"Stunt Racer 64") != NULL)
+     || (strstr((char*)rom_header.Name, (const char*)"GAUNTLET LEGENDS") != NULL)) {
```

Comment:
@bsmiles32 GLideN64 can now handle Gauntlet Legends in HLE so this can be removed